src/ex_docmd.c

```diff
@@ -1347,7 +1347,7 @@ do_cmdline(
 	    }
 	    else if (p != NULL)
 	    {
-		semsg(p);
+		semsg("%s", p);
```

Comment:
```suggestion
		emsg(p);
```

might be better?

Reply:
Thanks @k-takata.
I used your suggestion and fixed a couple of typos in comments in this file.